railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -25,6 +25,10 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
 <%- end -%>
 <% if RUBY_ENGINE == 'ruby' -%>
+
+# Reduces boot times through caching; configured in config/boot.rb
+gem 'bootsnap', '~> 1.0.0', platforms: [:mri]
```

Comment:
bootsnap only supports Ruby 2.3.0 and higher, but Rails still supports Ruby 2.2. 
https://github.com/Shopify/bootsnap/blob/master/bootsnap.gemspec#L26 

Should we check the Ruby version?